include/dxc/dxcpix.h

```diff
@@ -63,25 +63,25 @@ IDxcPixStructField : public IUnknown
       _Outptr_result_z_ BSTR *Name) = 0;
 
   virtual STDMETHODIMP GetType(
-      _Outptr_result_z_ IDxcPixType** ppType) = 0;
+      _COM_Outptr_ IDxcPixType** ppType) = 0;
 
   virtual STDMETHODIMP GetOffsetInBits(
-      _Outptr_result_z_ DWORD *pOffsetInBits) = 0;
+      _Out_ DWORD *pOffsetInBits) = 0;
 };
 
 struct __declspec(uuid("24c08c44-684b-4b1c-b41b-f8772383d074"))
 IDxcPixStructType : public IDxcPixType
 {
   virtual STDMETHODIMP GetNumFields(
-      _Outptr_result_z_ DWORD* ppNumFields) = 0;
+      _Out_ DWORD* ppNumFields) = 0;
```

Comment:
Not really related, but I notice this should be named pNumFields.

Reply:
Well spotted. I think I'll move on :-)